### ISSUE TITLE
Question about tutorial 1 and 2 Decoder

### ISSUE THREAD

**Issue by djaekim** [bentrevett/pytorch-seq2seq#168]
Hello, I had a question about
`prediction = self.fc_out(output)
        `

In the decoder in tutorial 2, why is the  `output = torch.cat((embedded.squeeze(0), hidden.squeeze(0), context.squeeze(0)), 
                           dim = 1)` as opposed to `output = torch.cat((embedded.squeeze(0), output.squeeze(0), context.squeeze(0)), 
                           dim = 1)`?

In tutorial 2 text, it says 
![image](https://user-images.githubusercontent.com/26092570/123689302-11dabb80-d821-11eb-8a05-ef225f6d746f.png)

Thank you!

**Comment by bentrevett** [comment]
When we have a sequence length of one, which we do when decoding, then `output == hidden`, as `output` is the hidden state from all time-steps, and the `hidden` is the hidden state from the final time-step, so if we have just a single time-step then the two are identical. So both of the code snippets in your question do the exact same thing.
